gui/music_line.hpp

```diff
@@ -4,7 +4,9 @@
 #include <iostream>
 #include <QWidget>
 #include "game_window.hpp"
-typedef std::string string;
```

Comment:
@emersion, are you the one who wrote this ?

Reply:
[`git blame`](https://github.com/fanfaresimulator/fanfaresimulator/blame/177c3c99fe1c46c5d37d15d30809265e22f12288/gui/music_line.hpp) 🤔

Reply:
I don't want to blame the author. That's amazing.

Reply:
`git praise`?

Reply:
![capture du 2017-12-06 14-43-26](https://user-images.githubusercontent.com/16472988/33664424-e19448f8-da93-11e7-8d43-9cc40bfbc27a.png)

:'(

Reply:
```shell
git config --global alias.praise blame
```

👍